Model/EventsForThirdPartyModules.php

```diff
@@ -73,6 +82,16 @@ class EventsForThirdPartyModules
                 ],
             ],
         ],
+        "checkMirasvitCreditAdminQuoteUsed" => [
+            "listeners" => [
+                [
+                    "module" => "Mirasvit_Credit",
+                    "sendClasses" => ["Mirasvit\Credit\Model\Config"],
```

Comment:
If we need just check if class exists and don't create an instance we can use "`checkClasses`" instead of "`sendClasses`"

Reply:
Fixed